lib/filewatch/discover.rb

```diff
@@ -0,0 +1,141 @@
+require_relative 'watched_file'
+
+module FileWatch
+  class Discover
+    attr_reader  :logger, :files, :wf_vars
+
+    def initialize(opts, loggr)
+      @logger = loggr
+      @watching = []
+      @exclude = []
+      @files = {}
+      set_ignore_older(opts[:ignore_older])
+      set_close_older(opts[:close_older])
+      @wf_vars = [opts[:delimiter], @ignore_older, @close_older]
```

Comment:
we tend to not abbreviate vars whenever possible, or at least document what the name stands for in their declaration, as to avoid forcing the reader to guess.